canonical/vanilla-framework
# Update chips with new theming support

## Done
- update chips with light and dark theme support
- add examples for dark theme version

## Fixes 
- https://warthogs.atlassian.net/browse/WD-7600

## QA
- Open [demo](https://vanilla-framework-4986.demos.haus/docs/examples/patterns/chip/variants-dark) and compare with [the existing version](https://vanillaframework.io/docs/examples/patterns/chip/variants) to check if chips components behave as expected

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
<img width="793" alt="Screenshot 2024-02-08 at 6 16 44 PM" src="https://github.com/canonical/vanilla-framework/assets/90341644/6c0ce07b-b124-4aed-82d4-9c755455da06">
